### ISSUE TITLE
Form 6 missing from Committee page 24/48 hr report section

### ISSUE THREAD

**Issue by lbeaufort** [fecgov/fec-cms#2194]
@llienfec commented on [Wed Jul 18 2018](https://github.com/fecgov/openFEC/issues/3289)

Form 6s are missing from the Committee pages in the 24 and 48 hour reports section.  

https://www.fec.gov/data/committee/C00606939/?tab=filings#notices

For testing: examples of committees that have Form 6s filed this cycle
PA) Lou Barletta for Senate – C00445122
(TX) Dan Crenshaw for Congress – C00660795
(SC) William Timmons for Congress – C00668491

**Comment by lbeaufort** [comment]
This committee's 48hr reports (form type F6) didn't have a report type. We were filtering by report_type = ['24', '48','-Q1', '-Q2', '-Q3', '-YE']. When the API searches for specific "include" values (24/48), it excludes null values. 

Example query on API side:
```
SELECT *  FROM ofec_filings_all_mv 
     WHERE (ofec_filings_all_mv.report_type NOT IN ('Q1', 'Q2', 'Q3', 'YE') OR ofec_filings_all_mv.report_type IS NULL) AND ofec_filings_all_mv.report_type IN ('24', '48') AND ofec_filings_all_mv.form_type IN ('F5', 'F24', 'F6', 'F9', 'F10', 'F11', 'RFAI', 'FRQ') AND (ofec_filings_all_mv.request_type NOT IN ('1', '3', '4', '5', '6', '7', '8', '9') OR ofec_filings_all_mv.request_type IS NULL) AND ofec_filings_all_mv.cycle IN (2018) AND ofec_filings_all_mv.committee_id = 'C00606939'
```

If we just filter by ['-Q1', '-Q2', '-Q3', '-YE'] we should still get the results we want.

**Comment by lbeaufort** [comment]
Addressed by [MERGE WITH GIT FLOW] Fix missing F6's on committee filings page #2195
